### PR TITLE
Loading: Add Class/Style to the mudprogress component, and style to text

### PR DESCRIPTION
**Pull request by Aerevia**

### CODE REVIEW

File: CodeBeam.MudExtensions/Components/Loading/MudLoading.razor [hunk 1]
```diff
@@ -30,8 +30,8 @@
         else
         {
 
-            <MudProgressLinear Class="mb-2" Color="Color.Primary" Indeterminate="true" />
-            <MudText Class="@TextClassname" Align="Align.Center">@Text</MudText>
+            <MudProgressLinear Class="@LoaderClassname" Style="@LoaderStylename" Color="Color.Primary" Indeterminate="true" />
```
Comment (mckaragoz) [inline CodeBeam.MudExtensions/Components/Loading/MudLoading.razor]: We can make it simpler, LoaderClassname does not have predefined class.

Can you change this like `Class="@ClassLoader"` and the other similar?

**Review by mckaragoz — CHANGES_REQUESTED** [review]

File: CodeBeam.MudExtensions/Components/Loading/MudLoading.razor (hunk 1 above)
Comment (Aerevia) [reply CodeBeam.MudExtensions/Components/Loading/MudLoading.razor]: I wanted to keep the "mb-2" class on Linear LoaderType but it doesn't seem necessary, i'll change it

File: CodeBeam.MudExtensions/Components/Loading/MudLoading.razor.cs [hunk 2]
```diff
@@ -64,6 +64,27 @@ public bool Loading
...
+        [Parameter]
+        [Category(CategoryTypes.FormComponent.Appearance)]
+        public string ClassLoader { get; set; }
+
+        /// <summary>
+        /// CSS style for the text.
+        /// </summary>
+        [Parameter]
+        [Category(CategoryTypes.FormComponent.Appearance)]
+        public string TextStyle { get; set; }
```
Comment (mckaragoz) [inline CodeBeam.MudExtensions/Components/Loading/MudLoading.razor.cs]: Sorry for another thing again, we should name `Class` and `Style` first like `StyleText`. So when user write Style, they show all parameters that about Style.

**Review by mckaragoz — CHANGES_REQUESTED** [review]

File: CodeBeam.MudExtensions/Components/Loading/MudLoading.razor.cs (hunk 2 above)
Comment (Aerevia) [reply CodeBeam.MudExtensions/Components/Loading/MudLoading.razor.cs]: fixed

**Comment by mckaragoz** [comment]
Thanks 🥇

**Pull request merged**